package.json

```diff
@@ -21,11 +21,13 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
+    "apexcharts": "^3.16.0",
     "ember-cli-babel": "^7.17.2",
     "ember-cli-htmlbars": "^4.2.2"
   },
   "devDependencies": {
     "@ember/optional-features": "^1.3.0",
+    "@ember/render-modifiers": "^1.0.2",
     "@glimmer/component": "^1.0.0",
     "@glimmer/tracking": "^1.0.0",
```

Comment:
These might also need to be in dependencies

Reply:
Can't select it for review but `ember-auto-import` should be a dependency for addons. After moving it to dependencies, can you try reverting your changes in `index.js` for manually import apexcharts to see if it worked?

Reply:
ok.. got it working, had to create an alias in index.js